core/vm/ipgraph.go

```diff
@@ -44,25 +81,25 @@ func (c *ipGraph) Run(evm *EVM, input []byte) ([]byte, error) {
 
 	switch {
 	case bytes.Equal(selector, addParentIpSelector):
-		return c.addParentIp(args, evm)
+		return c.addParentIp(args, evm, ipGraphAddress)
```

Comment:
@kingster-will Do we know why we need to add ipGraphAddress as separate parameters for all functions here? I don't see it changes any behavior?

Reply:
Yes, it is for reuse code between two ipgraph precomiple contract.